src/docs/truffle-dashboard/quickstart.md

```diff
@@ -1,16 +1,25 @@
 ---
-title: Use Truffle Dashboard
+title: Truffle Dashboard quickstart
 layout: docs.hbs
 ---
-# Use Truffle Dashboard
 
-When deploying your smart contracts you need to specify an Ethereum account that has enough funds to cover the transaction fees of the deployment. A popular method of doing this is copy-pasting your mnemonic phrase to a gitignored `.env` file so that it can be used for, e.g., the `@truffle/hdwallet-provider`. However, it is generally a bad practice to copy-paste your keys, especially since we have wallets like MetaMask that can send transactions for us.
+# Truffle Dashboard quickstart
 
-We developed the Truffle Dashboard to provide an easy way to use your existing MetaMask wallet for your deployments and for other transactions that you need to send from a command line context. Because the Truffle Dashboard connects directly to MetaMask it is also possible to use it in combination with hardware wallets like Ledger or Trezor.
+## Installation
+
+### As part of Truffle CLI
+
+Truffle Dashboard comes with Truffle CLI, so if you have a global Truffle CLI installation, you can run `truffle dashboard` in the console to start a dashboard instance. If you plan to use Truffle Dashboard with Hardhat, you'll probably want to use the npx approach unless you know you want to use Truffle CLI. Otherwise, [here are the installation instructions for Truffle CLI](/docs/truffle/how-to/install/).
+
+### Standalone with npx
+
+To run Truffle Dashboard without having to formally install Truffle CLI, you can run `npx truffle dashboard` and dependency installation will be handled for you.
+
+Note: Currently this approach still installs Truffle CLI in the background. We're working to remove dependencies on Truffle CLI, so in the future this command will get progressively lighter.
 
 ## Starting a dashboard
 
-To start a Truffle Dashboard, you need to run the `truffle dashboard` command in a separate terminal window.
+To start a Truffle Dashboard, you need to run the `truffle dashboard` command (or `npx truffle dashboard`) in a separate terminal window.
```

Comment:
I know this isn't something that you added but it feels weird to say "start _a_ Truffle Dashboard"